src/term/mod.rs

```diff
@@ -31,6 +31,8 @@ pub struct Book {
 
   /// A custom or default "main" entrypoint.
   pub entrypoint: Option<Name>,
+
+  pub info: Info,
```

Comment:
Although this is part of the compilation context, i don't think it's part of the internal representation of the program.

I understand that this makes all the passes simpler to write and call, but maybe there's a better alternative.

Reply:
Maybe having an struct `Ctx { Book, Info }`?

Reply:
Could be. You'll have to try it out and see how it turns outs.